lib/ansible/modules/cloud/google/gcp_compute_global_address.py

```diff
@@ -49,6 +49,11 @@
     - present
     - absent
     default: present
+  address:
```

Comment:
Should this be a ResourceRef?

Ansible is the only place (to my knowledge) where ResourceRefs do different things. I'm looking into trying to change this in the future, but it would be nice to use ResourceRefs properly for now.

If not, then I'm fine with this!

Reply:
Nah this is a straight string of an ip address. It's got nothing to do with an `Address`